### ISSUE TITLE
Action view for mass update

### ISSUE THREAD

**Issue by micker** [FLEXIcontent/flexicontent-cck#330]
It will be interesting to add an action function to update field with mass action
like acymailling action function.
ex
1 select item in list
2 clic on mass action field => open popup
3 select action 1 update field value, changing type, changing template, changing language
4 if we select changing field value select field in dropdown, if we select type action select template .. etc
5 after selecting a field type new value (or select value for radio, select etc)

6A execute action
6B Add an other action 

it will be good

**Comment by ggppdk** [comment]
Thinking of it

in a modal popup the 
- item form HTML of the field
would be displayed, with a submit button

Then we may go this further and display item form to massively update items
- every field will be activated via a checkbox or button
- then field value would be set
- then form would be submitted

**Comment by micker** [comment]
yep did you see that acymailling do
https://www.acyba.com/acymailing/148-acymailing-filter.html
![massactions](https://cloud.githubusercontent.com/assets/2539440/10765716/641e87ea-7cd4-11e5-8ec4-ece06b228ded.png)

enchance for later

**Comment by ggppdk** [comment]
Instead of showing filters inside the item fom
- you could use the filters in the items manager and then tick the list items
but i understand that when having many items this would need to be convenient

But then 
- we do have (a yet not public field) that displays field filters inside item form
so we could use the code if it to display filters inside item form for a different purpose (= for selecting which items to update)

**Comment by micker** [comment]
yep i think we have a good base for a futur buig enchanced (with app systeme)
